packages/apis/ipfs/src/query/schema.graphql

```diff
@@ -0,0 +1,6 @@
+#import { Query, ResponseType } into Http from "w3://ens/http.web3api.eth"
+
+type Query {
+  catToString(ipfsUrl: String!, cid: String!): String!
+  catFile(ipfsUrl: String!, cid: String!): Bytes!
```

Comment:
We can add `CatFileOptions` as another argument here. Some of the arguments I think would be useful (based on this: https://github.com/ipfs-shipyard/js-ipfs-http-client-lite/blob/HEAD/API.md#parameters-7) would be:
- headers
- offset
- length
- qs (query string)